packages/sdk/src/server/index.ts

```diff
@@ -22,6 +27,16 @@ export type CreateServerClientOpts = {
    */
   secretKey: string;
 
+  /**
+   * The client ID of your workspace's OAuth application.
+   */
+  oauthClientId: string;
```

Comment:
We should make these optional for now, and handle that optionality in the rest of the code, since project keys alone can authorize Connect API requests and we don't yet want to force those customers to use OAuth clients. We may end up consolidating on OAuth for the Connect API, too, and just make people pass the dev / prod project ID (removing project secret entirely), but for now it's an either-or.